File: s9/d60/lambda_node/lambda_function.js
exports.handler = async (event, context) => {
    console.log('Event:', JSON.stringify(event,null,2));
    const response = {
        statusCode: 200,
        headers:{
            'Content-Type': 'application/json'
        },
        body: JSON.stringify({
            message: '¡Hola desde Node.js Lambda!',
            input: event,
            timestamp: new Date().toISOString(),
            nodeVersion: process.version
        })
    };

    console.log('Response:',JSON.stringify(response,null,2));
    return response;
};

File: s9/d60/lambdas/lambda_function.py
import json
import datetime

def lambda_handler(event,context):
    """
    Función lambda basica que recibe un evento y retorna una respuesta
    """
    print(f"Evento recibido: {json.dumps(event)}")
    print(f"Context: {context}")

    # Procesar el evento
    name = event.get('name','Roxs Developer')
    message = event.get('message','Hello from localstack lambda')

    # Crear respuesta
    response = {
        'statusCode': 200,
        'headers':{
            'Content-Type': 'application/json'
        },
        'body': json.dumps({
            'message': f"¡Hola {name}!",
            'input_message': message,
            'timestamp': datetime.datetime.now().isoformat(),
            'function_name': context.function_name if hasattr(context, 'function_name') else 'roxs-function',
            'processed_by': 'LocalStack Lambda'
        })
    }

    print(f"Respuesta: {json.dumps(response)}")
    return response